src/js/view/controls/controls.js

```diff
@@ -200,34 +205,23 @@ export default class Controls extends Events {
                     api.play(settingsInteraction);
                 }
             }
-
-            // Trigger userActive so that a dismissive click outside the player can hide the controlbar
-            const activeTimeout = (visible || isKeyEvent) ? 0 : ACTIVE_TIMEOUT;
-            this.userActive(activeTimeout);
-            lastState = state;
-
-            const settingsButton = this.controlbar.elements.settingsButton;
-
-            if (!visible && settingsButton) {
-                if (isKeyEvent) {
-                    settingsButton.element().focus();
-                } else if (evt) {
-                    focusFloatingElement();
-                }
+            if (!visible && isKeyEvent && settingsButton) {
+                settingsButton.element().focus();
             }
-        };
-        const settingsMenu = this.settingsMenu = createSettingsMenu(
-            controlbar,
-            visibilityChangeHandler,
-            model.get('localization')
-        );
-        setupSubmenuListeners(settingsMenu, controlbar, model, api);
+        });
+        settingsMenu.on('menuVisibility', (menu) => this.controlbar.trigger('menuVisibility', menu));
```

Comment:
Why does this event need to be forwarded through the `controlbar`?

It looks like you are only listening on the control bar above, so why not just listen on `settingsMenu` up there?